@claude - getting:
```
error[E0502]: cannot borrow `cmd` as immutable because it is also borrowed as mutable
   --> /home/runner/work/shards/shards/shards/lang/src/cli.rs:161:29
    |
161 |   generate(shell, &mut cmd, cmd.get_name(), &mut std::io::stdout());
    |   --------        --------  ^^^ immutable borrow occurs here
    |   |               |
    |   |               mutable borrow occurs here
    |   mutable borrow later used by call
 ```

--- a/shards/lang/src/cli.rs
+++ b/shards/lang/src/cli.rs
@@ -158,7 +158,8 @@ struct SimpleCLI {
 
 fn generate_completions(shell: Shell) {
   let mut cmd = Cli::command();
-  generate(shell, &mut cmd, cmd.get_name(), &mut std::io::stdout());
+  let name = cmd.get_name().to_string();
+  generate(shell, &mut cmd, name, &mut std::io::stdout());
 }
 
 pub fn process_args(argc: i32, argv: *const *const c_char, no_cancellation: bool) -> i32 {

